substrate/frame/recovery/src/lib.rs

```diff
@@ -178,19 +181,20 @@ mod mock;
 mod tests;
 pub mod weights;
 
+type AccountIdLookupOf<T> = <<T as frame_system::Config>::Lookup as StaticLookup>::Source;
 type BalanceOf<T> =
 	<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
-
+type BlockNumberFromProvider<T> =
```

Comment:
Not fully convinced by this name

```suggestion
type BlockNumberFromProviderOf<T> =
```
Seems to me to fit better with the convention